graphql/admin/admin.go

```diff
@@ -426,81 +425,25 @@ func newAdminResolverFactory() resolve.ResolverFactory {
 				})
 		}).
 		WithMutationResolver("export", func(m schema.Mutation) resolve.MutationResolver {
-			export := &exportResolver{}
-
-			// export implements the mutation rewriter, executor and query executor hence its passed
-			// thrice here.
-			return resolve.NewMutationResolver(
-				export,
-				export,
-				export,
-				resolve.StdMutationCompletion(m.ResponseName()))
+			return resolve.MutationResolverFunc(resolveExport)
```

Comment:
could this be taken one level further?

const MutationResolvers = [
  ["export", resolveExport]
...
]

then iterate through these factory = factory.WithMutationResolver(name, ...)

Reply:
Yeah, nice idea.

Done for the ones I've changed in this PR. Will move more into that sort of setup as I clean up more of the functions.